content/contributing/development/ui/scss.rst

```diff
@@ -0,0 +1,259 @@
+
+.. _contributing/scss:
+
+===============
+SCSS Guidelines
+===============
+
+
+
+Syntax and Formatting
+=====================
+
+.. tabs::
+
+    .. code-tab:: html SCSS
+
+        .o_foo, .o_foo_bar,
+        .o_baz {
+            height: $o-statusbar-height;
+
+            .o_qux {
+                height: $o-statusbar-height * 0.5;
+            }
+        }
+
+        .o_corge {
+            background: $o-list-footer-bg-color;
+        }
+
+    .. code-tab:: css CSS
+
+        .o_foo, .o_foo_bar, .o_baz, .o_qux {
+            height: 32px;
+        }
+
+        .o_foo .o_quux, .o_foo_bar .o_quux, .o_baz .o_quux {
+            height: 16px;
+        }
+
+        .o_corge {
+            background: #EAEAEA;
+        }
+
+* four (4) space indents, no tabs;
+* 80 character wide columns;
+* related selectors on the **same** line; unrelated selectors on new lines;
+* empty space between the last selector and the opening brace (`{`).
+* the closing brace (`}`) on its own new line.
+* one line for each declaration;
+* meaningful use of whitespace.
```

Comment:
- Not a requirement, but we prefer using `-` instead of `*` for the lists
- 80 characters? 80-character?